src/scripts/process_us_ca.py

```
@@ -128,71 +148,95 @@ def save_data(data):
+                break
+            if len(item["text"]) < 200:
+                continue  # 너무 짧은 건 버림
+
+            rows = save_data({"text": item["text"], "country": "US"})
             final_data.extend(rows)
             count += 1
-            if count % 50 == 0: 
+            if count % 50 == 0:
                 print(f"   Running... {count} (Sample Title: {rows[0]['law_title']})")
-    except Exception as e: print(f"US Error: {e}")
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)